tests/cbmc/proofs/s2n_free/s2n_free_harness.c

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include "api/s2n.h"
+#include "utils/s2n_blob.h"
+#include "error/s2n_errno.h"
+
+#include <assert.h>
+#include <cbmc_proof/proof_allocators.h>
+#include <cbmc_proof/make_common_datastructures.h>
+#include <cbmc_proof/cbmc_utils.h>
+
+int munlock(const void *addr, size_t len)
+{
+    assert(S2N_MEM_IS_WRITABLE(addr,len));
+    return nondet_int();
+}
+
+int s2n_calculate_stacktrace() { return nondet_int(); }
```

Comment:
CBMC actually have a set of stubs that could be used across all proofs. Take a look at `s2n/tests/cbmc/stubs`. So you can remove this `s2n_calculate_stacktrace` from the harness and add it to the Makefile as
```Makefile
ABSTRACTIONS += $(HELPERDIR)/stubs/s2n_calculate_stacktrace.c
```

And you could do the same for `munlock` and `sysconf`, but you need to add the `.c` files for them 
```Makefile
ABSTRACTIONS += $(HELPERDIR)/stubs/munlock.c
ABSTRACTIONS += $(HELPERDIR)/stubs/sysconf.c
```

Reply:
Added